src/DssSpell.t.sol

```diff
@@ -545,4 +534,126 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(gov.balanceOf(wallets.addr("RWF_WALLET")),  prevMkrRWF    + amountRWF);
         assertEq(gov.balanceOf(wallets.addr("DECO_WALLET")), prevMkrDeco   + amountDeco);
     }
+
+    // All times in GMT
+    uint256 constant JUL_01_2022 = 1656633600; // Friday,   July      1, 2022 12:00:00 AM
+    uint256 constant OCT_01_2022 = 1664582400; // Saturday, October   1, 2022 12:00:00 AM 
+    uint256 constant OCT_31_2022 = 1667260799; // Monday,   October  31, 2022 11:59:59 PM
+    uint256 constant JUN_30_2023 = 1688169599; // Friday,   June     30, 2023 11:59:59 PM
+    uint256 constant AUG_31_2023 = 1693526399; // Thursday, August   31, 2023 11:59:59 PM
+    uint256 constant DEC_31_2022 = 1672531199; // Saturday, December 31, 2022 11:59:59 PM
+
+    function testDaiStreams() public { // make private to disable
```

Comment:
Consider using the existing testVestDAI instead of this new test. We can still call `checkDaiVest` from there instead of the inlined checks if we want, but we probably don't want to leave the code of the 2 tests which basically check the same things.

Reply:
Good find Tal.  I missed the existing test template when I looked for it, so I dragged this out of the archive and assumed it was right because `checkDaiVest()` was still in the base.  I've adapted the existing `testVestDAI()` template to use `checkDaiVest()` rather than the inline checks, and I also updated the later to include the `days` check on `fin`.